### ISSUE TITLE
Fixer changed his API

### ISSUE THREAD

**Issue by dudztroyer** [AlessandroMinoccheri/codeigniter-currency-converter#8]
Fixer updated his API and now it does not work anymore with the current setup. 

Need some modifications in order to make it work, like adding an API key and such. Can you do it?

I'll try to make a working version, but in the meanwhile I think you should know that it does not work.

**Comment by AlessandroMinoccheri** [comment]
I will check it in these days. Thanks to have opened this issue

**Comment by dudztroyer** [comment]
I did a quick fix using another provider. 
Here is the getRates() function to make it work as it was. Just commented out the query to fixer in order to show the differences if anything goes wrong.

````
    private function getRates()
    {
        //$url = 'http://api.fixer.io/latest?base=' . $this->fromCurrency . '&symbols=' . $this->toCurrency;
        $url = 'https://free.currencyconverterapi.com/api/v5/convert?q=' . $this->fromCurrency . '_' . $this->toCurrency . '&compact=ultra' ;
        $handle = @fopen($url, 'r');

        if ($handle) {
            $result = fgets($handle, 4096);
            fclose($handle);
        }

        if (isset($result)) {
            $conversion = json_decode($result, true);

           /* if (isset($conversion['rates'][$this->toCurrency])) {
                return $conversion['rates'][$this->toCurrency];
            }*/
			if (isset($conversion[$this->fromCurrency . '_' . $this->toCurrency])) {
                return $conversion[$this->fromCurrency . '_' . $this->toCurrency];
            }
        }

        return $this->rate = 0;
    }
````

I'll be waiting the official version from you, since I do not know how reliable this service is hehe

**Comment by dudztroyer** [comment]
Also, note that the version in the library file is different from the version said in the readme. 
Should be updated in order to avoid versioning conflict. 

Thanks

**Comment by AlessandroMinoccheri** [comment]
Ok I have seen the problem, now fixer has changed his policy and pricing, we can change the service to https://free.currencyconverterapi.com as you have suggest, what do you think?

**Comment by AlessandroMinoccheri** [comment]
Ok, I have compared fixer and free.currencyconverter and for a requests reason I think we could move to free.currencyconverter, would you like to open a Pull Request with your code? Be sure that tests are green @dudztroyer

**Comment by dudztroyer** [comment]
Well, it is already on production side... So it should be ok. https://www.wsbot.net/purchase

I do not really mess with github, so I prefer avoiding it. I'll try to do it, but if anything, just dont approve and change by yourself hehe

**Comment by AlessandroMinoccheri** [comment]
It's not a problem if you don't really mess with github, we can learn together.
This could be a good way to improve skills each other, so feel free to complete the pull request @dudztroyer

**Comment by AlessandroMinoccheri** [comment]
Resolved into pull request #9
